README.md

```diff
@@ -14,6 +14,24 @@
 
 Download pre-compiled binaries for Mac, Windows and Linux from the [releases](https://github.com/dropbox/dbxcli/releases) page.
 
+### Linux Installation of pre-compiled binaries
+1. Check whether your CPU is compatible with AMD/Intel or ARM
+```sh
+uname -p
+```
+If the result reads `x86_64` use the `AMD64` binary.  We will continue assuming you have x86_64 architecture (most common).
+
+2. Download the `dbxcli` binary into your binary folder and rename it to `dbxcli`
```

Comment:
Everything below this can be the same for Mac and Linux. I would like to avoid creating the perception that you need to use `sudo` to be able to download and use `dbxcli` (you don't).

Reply:
You need to use `sudo` to change the ownership of the binary to an executable.  Is there a way to do that without sudo?  Took me a good 30 min to figure that one out.  Agreed that we can merge instructions for Linux and Mac

Reply:
> "You need to use sudo to change the ownership of the binary to an executable. Is there a way to do that without sudo?"

You definitely don't need sudo. In your case, I suspect what happened is that you had to use sudo because the binary was placed in `/bin`. If you download to a folder you already own (e.g. `$HOME/bin`), you shouldn't need sudo either to rename or to make executable.